migrations/R__create_session_id_index_on_audit_events.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX CONCURRENTLY IF NOT EXISTS audit_events_session_id_idx ON audit.audit_events (session_id);
```

Comment:
does this need to be `audit_events_session_id_idx1`

Reply:
No, I don't think so. We do have some others without a numeric suffix. Not sure why some do and some don't. We were originally just using the name it already had in the DB, but then we renamed it anyway, so that argument doesn't really hold anymore... -.-

Reply:
Do you think it _should_ have the suffix? If so, can I ask why? It's not like there's an `index2` or even that we're likely to create another index using a different algorithm (which is about the only use case I can think of that you'd need a disambiguator). So I'm not sure what benefit the `1` would serve... 😕